include/ibamr/IBFESurfaceMethod.h

```diff
@@ -581,6 +577,44 @@ class IBFESurfaceMethod : public IBStrategy
      */
     std::unique_ptr<IBTK::LibMeshSystemIBVectors> d_X_vecs;
 
+    /*!
+     * Object managing access to libMesh system vectors for the velocity system.
+     *
+     * @note Unlike IBAMR::IBFEMethod, this class does not inherit from
+     * FEMechanicsBase and therefore both normal and IB vectors are handled by
+     * d_U_vecs.
+     */
+    std::unique_ptr<IBTK::LibMeshSystemIBVectors> d_U_vecs;
+
+    /*!
+     * Object managing access to libMesh system vectors for the normal velocity
```

Comment:
Here's a question --- do we still need all of those different systems, or could we just have `U`, `U_n`, and `U_t` system vectors?

Reply:
We had different systems originally because my vector management "system" was terrible.

Reply:
I think at some point we decided that it was useful to visualize these things, which means they really do need their own `System`.